assets/js/components/notifications/SwitchedToGA4Banner.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * Site Kit by Google, Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import GA4SuccessGreenSVG from '../../../svg/graphics/ga4-success-green.svg';
+import { CORE_USER } from '../../googlesitekit/datastore/user/constants';
+import { CORE_UI } from '../../googlesitekit/datastore/ui/constants';
+import { MODULES_ANALYTICS } from '../../modules/analytics/datastore/constants';
+import { UA_CUTOFF_DATE } from '../../modules/analytics/constants';
+import { stringToDate } from '../../util';
+import ga4Reporting from '../../feature-tours/ga4-reporting';
+import BannerNotification from './BannerNotification';
+const { useSelect, useDispatch } = Data;
+
+export default function SwitchedToGA4Banner() {
+	const isGA4DashboardView = useSelect( ( select ) =>
+		select( MODULES_ANALYTICS ).isGA4DashboardView()
+	);
+
+	const referenceDate = useSelect( ( select ) =>
+		select( CORE_USER ).getReferenceDate()
+	);
+
+	const isTourDismissed = useSelect( ( select ) =>
+		select( CORE_USER ).isTourDismissed( ga4Reporting.slug )
+	);
+
+	const showGA4ReportingTour = useSelect( ( select ) => {
+		return select( CORE_UI ).getValue( 'showGA4ReportingTour' );
+	} );
+
+	const { setValue } = useDispatch( CORE_UI );
+	const handleCTAClick = () => {
+		setValue( 'showGA4ReportingTour', true );
+	};
+
+	const { dismissTour } = useDispatch( CORE_USER );
+	const handleDismissClick = () => {
+		dismissTour( ga4Reporting.slug );
+	};
+
+	if ( ! isGA4DashboardView || isTourDismissed || showGA4ReportingTour ) {
+		return null;
+	}
```

Comment:
We already check this in the `BannerNotifications` component when we render the `SwitchedToGA4Banner` notification. We verify that the Analytics module is connected, which means that property ID is set.

Reply:
Ah, yes, just noticed, thank you!

However, do you think it is sufficient to use `isModuleConnected` to check if UA is enabled? I ask because we have changed the connection logic for the Analytics module and it would report it as connected even though UA is not. Screenshot:

![image](https://github.com/google/site-kit-wp/assets/20284937/b9f7bcd0-4bfa-48d5-a228-a462982e5f4b)

Do you think it'd be wiser to check if UA has a property ID instead to verify this? Thanks!

Reply:
I am not sure why we would need UA property ID tbh. Looks like we don't rely on it, I think we should be good.

Reply:
Thank you, @eugene-manuilov.

Based on this confirmation from Evan and our internal discussions, I have added an additional condition here to check if UA is connected.